### ISSUE TITLE
Don't create a new draft/reviewed version if "save and continue editing" is clicked

### ISSUE THREAD

**Issue by andreash** [jgarber/radiant-chronicle-extension#33]
If I intend "just to save" and then continue my work, it might be a good idea, that no new version is created, at least not if the page isn't published yet. How about just saving the updated page into the current version?

Cheers,
Andreas

**Comment by jgarber** [comment]
Yeah, I agree.  I think we're going to have 100 drafts between published versions if we don't.  The only thing I worry about is how it affects rollback.  If I keep making a change and clicking Save and Continue Editing but then I accidently delete a chunk of text, there's no way to go back.

**Comment by jgarber** [comment]
So I'm thinking there might be a way to group minor revisions by some combination of time, button clicked, and status.

**Comment by nicholasjhenry** [comment]
I commented about this in Issue #11, but it appears my comments maybe more relevant here:

An alternative implementation could be a "Save without Revision" option which overwrites the last revision (similar to Writeboard)
